Makefile.core.mk

```diff
@@ -68,6 +68,9 @@ NAMESPACE ?= istio-operator
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION = 1.26.0
 
+# Set DOCKER_BUILD_FLAGS to specify flags to pass to 'docker build', default to empty. Example: --platform=linux/arm64
+DOCKER_BUILD_FLAGS ?= "--platform=$(TARGET_OS)/$(TARGET_ARCH)"
```

Comment:
we don't seem to set TARGET_OS and TARGET_ARCH by default- should we?

Reply:
is set by this script by default `/common/scripts/setup_env.sh`

Reply:
oh I see! thanks, nevermind then